hack/ginkgo-e2e.sh

```diff
@@ -103,8 +103,16 @@ else
   auth_config=()
 fi
 
+if [[ "$KUBERNETES_PROVIDER" == "libvirt-coreos" ]]; then
```

Comment:
Can you just redefine the host argument instead of duplicating the entire command line? It will make it easier to see what the delta is for the libvirt-coreos provider.